.github/workflows/kind-e2e.yaml

```diff
@@ -37,8 +37,39 @@ jobs:
   # However, KinD currently does not support Windows nodes so this is
   # not intended to be a complete replacement for our AKS-based E2E
   # workflows.
+  build-e2e:
+    name: e2e image build
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY_PORT: 5000
+      REGISTRY_NAME: kind-registry
+      DAPR_REGISTRY: localhost:5000/dapr
+      DAPR_TAG: dev
+      DAPR_NAMESPACE: dapr-tests
+      GOVER: 1.17
+    steps:
+    - name: Set up Go ${{ env.GOVER }}
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GOVER }}
+    - name: Check out code onto GOPATH
+      uses: actions/checkout@v2
+      with:
+        path: ./src/github.com/dapr/dapr
+    # setup image cache
+    - uses: satackey/action-docker-layer-caching@v0.0.11
```

Comment:
We have attempted this in the past. Trouble was the Windows images being extremely large ended up paving the cache and breaking this optimization. But, let's give it a try and see what happens.

Reply:
It is for kind e2e test which only run at Linux I think.